tilequeue/utils.py

```diff
@@ -23,3 +26,51 @@ def grouper(iterable, n):
         if not chunk:
             return
         yield chunk
+
+def parse_log_file(log_file):
+    ip_pattern = '(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
+    # didn't match againts explicit date pattern, in case it changes
+    date_pattern = '\[([\d\w\s\/:]+)\]'
+    tile_id_pattern = '\/([\w]+)\/([\d]+)\/([\d]+)\/([\d]+)\.([\d\w]*)'
+
+    log_pattern = '%s - - %s "([\w]+) %s.*' % (ip_pattern, date_pattern, tile_id_pattern)
+    
+    matches = filter(
+        lambda match: match and len(match.groups()) == 8, 
+        map(lambda log_string: re.search(log_pattern, log_string), log_file)
+    )
+
+    iped_dated_coords = map(lambda match: (match.group(1), 
+                                           datetime.strptime(match.group(2), '%d/%B/%Y %H:%M:%S'), 
+                                           coord_marshall_int(create_coord(match.group(6), match.group(7), match.group(5)))), matches)
```

Comment:
addressed in https://github.com/tilezen/tilequeue/pull/204/commits/e95e234d89c5531f46582ff3833743e31703061b